src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -130,11 +130,16 @@ g.test('mipLevelCount,format')
     const info = kTextureFormatInfo[format];
     await t.selectDeviceOrSkipTestCase(info.feature);
 
+    // Compute dimensions such that the dimensions are less than 2^6 (32) but also a multiple of the format block size.
```

Comment:
nit: More precisely I think what you're trying to get is a size between 17 and 32 inclusive (so there will be exactly 6 mip levels). So perhaps comment that and assert afterward than the results are in that range.

Reply:
Done.